### ISSUE TITLE
Certbot cant find the inwx plugin even though the installation seems successfull

### ISSUE THREAD

**Issue by flokoe** [oGGy990/certbot-dns-inwx#3]
Hey,

first I want to thank you. I really like inwx and I really appreciate that your wrote an certbot plugin for inwx dns verification (I need a wildcard certificate and for this I need dns verification).
Unfortunately certbot cant find the inwx plugin.

After the installation of the inwx plugin I execute the following command 
`certbot certonly -a certbot-dns-inwx:dns-inwx -d *.mydomain.com --dry-run`

and get the following error message:
```
Saving debug log to /var/log/letsencrypt/letsencrypt.log
Could not choose appropriate plugin: The requested certbot-dns-inwx:dns-inwx plugin does not appear to be installed
The requested certbot-dns-inwx:dns-inwx plugin does not appear to be installed
```

Here are some information about my system:
OS: Debian Stretch 9.4
Python 2: Version 2.7.13
Python 3: Version 3.5.3
Certbot: Verison 0.23.0 (from the stretch backports repo)
python-acme: 0.22.2 (from the stretch backports repo)

I am not a python expert, but I do have some experience in programming and I am working as an IT specialist so please do not hesitate to ask.
I am more then happy to provide any information you need.

Best regards
Florian

**Comment by oGGy990** [comment]
Hey,

have you watched out to run the setup via Python 3? Certbot is only using Python 3 nowadays, but iirc you can still install it in a Python 2 environment via pip by accident.

**Comment by oGGy990** [comment]
I have updated the README with some improved installation instructions (which I also learned the hard way :-) ). See 3dbd249.

**Comment by flokoe** [comment]
Hey,

thank you for your quick reply :)

You were right I totally over looked the fact, that certbot uses Python 3.

I used the command `python setup.py install` to install the inwx plugin. Unfortunately on most Linux distributions the command `python` defaults to `python2.7` (symlink), so the plugin got installed for the python 2.7 namespace.

I installed the plugin with `python3 setup.py install` again and now everything works as expected.  :)
Maybe you want to note this in your README.md

Thank your for your time and the hint with Python 3.
I think we can close this.

Best regards
Florian
